proto/dataplane/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+
+proto_library(
+    name = "dataplane_proto",
+    srcs = ["dataplane.proto"],
+    visibility = ["//visibility:public"],
+    deps = ["//proto/forwarding:forwarding_proto"],
+)
+
+go_proto_library(
+    name = "dataplane_go_proto",
+    importpath = "github.com/openconfig/lemming/proto/dataplane",
+    proto = ":dataplane_proto",
+    visibility = ["//visibility:public"],
+    deps = ["//proto/forwarding"],
+)
+
+go_library(
+    name = "dataplane",
+    srcs = ["generate.go"],
+    embed = [":dataplane_go_proto"],
+    importpath = "github.com/openconfig/lemming/proto/dataplane",
+    visibility = ["//visibility:public"],
+)
```

Comment:
This looks like a duplicate of `dataplane_go_proto`?

Reply:
they are actually subtly different, these rules are generated automatically.